### ISSUE TITLE
Unable to get metrics on custom data

### ISSUE THREAD

**Issue by SIDD-1991** [WongKinYiu/YOLO#164]
Hi

I currently have a custom dataset which i have organized like this 

**MAIN_FOLDER**/yolo/data/tmp/images->train(contains images for training )
**MAIN_FOLDER**/yolo/data/tmp/images->val(contains images for validation)

**MAIN_FOLDER**/yolo/data/tmp/labels->train(contains labels for training )
**MAIN_FOLDER**/yolo/data/tmp/labels->val(contains labels for training )

And below is the contents of my custom file general.yaml which i have placed in:
**MAIN_FOLDER**/yolo/config/dataset

Below is how my general.yaml looks like 

path: **MAIN_FOLDER**/yolo/data/tmp
train: train
validation: val
class_num: 2
class_list: ['3G', '4G']
image_size: [640, 640]
learning_rate: 0.0000001
out_path: runs
exist_ok: True
lucky_number: 10
use_wandb: True
use_tensorboard: False

weight: True # Path to weight or True for auto, False for no pretrained weight

The command i use for training is **python lazy.py task=train model=v9-m dataset=general device=gpu weight=False**

below is my training config file train.yaml

task: train

defaults:
  - validation: ../validation

epoch: 10

data:
  batch_size: 5
  image_size: ${image_size}
  cpu_num: ${cpu_num}
  shuffle: True
  pin_memory: True
  data_augment:
    Mosaic: 1
    # MixUp: 1
    # HorizontalFlip: 0.5
    RandomCrop: 1
    RemoveOutliers: 1e-8

optimizer:
  type: SGD
  args:
    lr: 0.00001
    weight_decay: 0.0005
    momentum: 0.937
    nesterov: true

loss:
  objective:
    BCELoss: 0.5
    BoxLoss: 7.5
    DFLoss: 1.5
  aux:
    0.25
  matcher:
    iou: CIoU
    topk: 10
    factor:
      iou: 6.0
      cls: 0.5

scheduler:
  type: LinearLR
  warmup:
    epochs: 3.0
  args:
    total_iters: ${task.epoch}
    start_factor: 1
    end_factor: 0.01

ema:
  enable: true
  decay: 0.995

Any idea what i am doing wrong maybe it isn't even loading my data.
